app/controllers/controllers.js

```diff
@@ -201,58 +201,74 @@ exports.showAllTeams = (req, res) => {
 
 // Handles teammate evaluation
 exports.evaluateTeammate = async (req, res) => {
-  if (req.session.user) {
-    var teammateID = req.params.id;
-    var reviewerID = req.session.user.id;
+  // Check if user is logged in (session should exist)
+  if (!req.session.user || !req.session.user.id) {
+    return res.redirect("/login"); // Redirect to login page if user is not logged in
+  }
+
+  const teammateID = req.params.id;
+  const reviewerID = req.session.user.id;
 
-    var sql =
-      `SELECT * FROM evaluations WHERE teammateID = ${teammateID} ` +
-      `AND reviewerID = ${reviewerID}`;
```

Comment:
use parameterized queries or prepared statements, which separate the data from the SQL query and ensure that user input is properly sanitized.